openquake/commonlib/calculators/hazard.py

```diff
@@ -222,20 +206,6 @@ def _init_tags(self):
         self.tag_seed_pairs = [(tag, rnd.randint(0, calc.MAX_INT))
                                for tag in self.tags]
 
-    def pre_execute(self):
-        """
-        Read the site collection and initialize GmfComputer, tags and seeds
-        """
-        if 'exposure' in self.oqparam.inputs:
-            logging.info('Reading the exposure')
-            exposure = readinput.get_exposure(self.oqparam)
-            logging.info('Reading the site collection')
-            self.sitecol, _assets = readinput.get_sitecol_assets(
-                self.oqparam, exposure)
-        else:
-            self.sitecol = readinput.get_site_collection(self.oqparam)
-        self._init_tags()
```

Comment:
This was removed since it is already done inside the HazardCalculator.pre_execute